scripts/us_census/acs5yr/subject_tables/s2201/process.py

```diff
@@ -0,0 +1,236 @@
+        elif p == 'Other family:' and (
+                'Male householder, no spouse present' in s or
+                'Female householder, no spouse present' in s):
+            continue
+        elif p == 'Other family' and (
+                'Male householder, no wife present' in s or
+                'Female householder, no husband present' in s):
+            continue
+        elif p == 'One race' and (
+                'White' in s or 'Black or African American' in s or
+                'American Indian and Alaska Native' in s or 'Asian' in s or
+                'Native Hawaiian and Other Pacific Islander' in s or
+                'Some other race' in s):
+            continue
+
+        elif p in _FEATURE_TO_PROPERTY:
+
+            # Work status is only provided for FamilyHousehold
+            if ((p == 'No workers in past 12 months' or
+                 p == '1 worker in past 12 months' or
+                 p == '2 or more workers in past 12 months') and
+                    'Families' not in s):
+                sv.append(_FEATURE_TO_PROPERTY['Families'])
+            sv.append(_FEATURE_TO_PROPERTY[p])
+    return '_'.join(sv)
+
+
+def create_csv(output, stat_vars):
+    """Creates output CSV file."""
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    with open(output, 'w') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        writer.writeheader()
+
+
+def write_csv(filename, reader, output, stat_vars):
+    """Reads input_file and writes cleaned CSV to output."""
+    if 'ACSST5Y' not in filename:
+        return
+    fieldnames = ['observationDate', 'observationAbout'] + stat_vars
+    stat_var_set = set(stat_vars)
+    with open(output, 'a') as f_out:
+        writer = csv.DictWriter(f_out, fieldnames=fieldnames)
+        observation_date = filename.split('ACSST5Y')[1][:4]
+        valid_columns = {}
+        for row in reader:
+            if row['GEO_ID'] == 'id':
+
+                # Map feature names to stat vars
+                for c in row:
+                    sv = convert_column_to_stat_var(row[c])
+                    if sv in stat_var_set:
+                        valid_columns[c] = sv
+                continue
+
+            new_row = {
+                'observationDate': observation_date,
+                'observationAbout': 'dcid:geoId/' + row['GEO_ID'].split('US')[1]
+            }
+            for c in row:
+
+                # We currently only support the stat vars in the list
+                if c not in valid_columns:
+                    continue
+                sv = valid_columns[c]
+
+                # Exclude missing values
+                if (row[c] == '**' or row[c] == '-' or row[c] == '***' or
+                        row[c] == '*****' or row[c] == 'N' or row[c] == '(X)' or
+                        row[c] == 'null'):
+                    continue
+
+                # Exclude percentages
+                if '.' in row[c]:
```

Comment:
Are decimals there only in percentages? My vague memory is Estimates (population counts) can be decimal too

Reply:
estimates can also be percentages (example: https://data.census.gov/cedsci/table?q=S2601A&tid=ACSST1Y2019.S2601A) but we are excluding these for now (and only including the raw numbers)

I haven't seen any cases where counts can be decimals, but can update the script if so